vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -127,214 +149,168 @@ private void initTagAndNotesLayout() {
+    getContent().addTitle(experiment.getName());
+    loadTagInformation(experiment);
+    loadExperimentInfo(experiment);
+    loadBlockingVariableInformation();
+    addVariablesDialog.experimentId(experiment.experimentId());
+    fillExperimentalGroupDialog();
+    loadExperimentalGroups();
+    if (experiment.variables().isEmpty()) {
+      useCaseNoVariablesYet();
+    } else {
+      removeDisclaimer();
+      displayExperimentalGroupsCollection();
     }
+  }
 
-    private void fillExperimentalGroupDialog() {
-      Objects.requireNonNull(experimentId, "experiment id not set");
-      List<ExperimentalVariable> variables = experimentInformationService.getVariablesOfExperiment(
-          experimentId);
-      List<VariableLevel> levels = variables.stream()
-          .flatMap(variable -> variable.levels().stream()).toList();
-      experimentalGroupsDialog.setLevels(levels);
-    }
+  private void useCaseNoVariablesYet() {
+    displayDisclaimer();
+    hideExperimentalGroupsCollection();
+  }
 
-    private void loadExperimentalGroups() {
-      Objects.requireNonNull(experimentId, "Experiment id not set");
-      // We load the experimental groups of the experiment and render them as cards
-      List<ExperimentalGroupCard> experimentalGroupsCards = experimentInformationService.experimentalGroupsFor(
-          experimentId).stream().map(ExperimentalGroupCard::new).toList();
+  private void loadTagInformation(Experiment experiment) {
+    tagLayout.removeAll();
+    List<String> tags = new ArrayList<>();
+    experiment.getSpecies().forEach(species -> tags.add(species.value()));
+    experiment.getSpecimens().forEach(specimen -> tags.add(specimen.value()));
+    experiment.getAnalytes().forEach(analyte -> tags.add(analyte.value()));
+    tags.forEach(tag -> tagLayout.add(new Tag(tag)));
+    tagLayout.getElement().setAttribute("Title", String.join(" ", tags));
+  }
 
-      // We register the experimental details component as listener for group deletion events
-      experimentalGroupsCards.forEach(this::subscribeToDeletionClickEvent);
-      experimentalGroupsCollection.removeAll();
-      experimentalGroupsCollection.addComponents(experimentalGroupsCards);
-    }
+  private void loadExperimentInfo(Experiment experiment) {
+    ExperimentInfoComponent factSheet = ExperimentInfoComponent.create(experiment.getSpecies(),
+        experiment.getSpecimens(), experiment.getAnalytes());
+    ExperimentalVariablesComponent experimentalVariables = ExperimentalVariablesComponent.create(
+        experiment.variables());
+    ExperimentDetailsComponent.this.experimentSummary.add(factSheet);
+    ExperimentDetailsComponent.this.experimentSummary.add(experimentalVariables);
+    factSheet.showMenu();
+  }
 
-    private void displayDisclaimer() {
-      contentExperimentalGroupsTab.add(noExperimentalVariablesDefined);
-    }
+  private void loadBlockingVariableInformation() {
+    //ToDo load information from backend once implemented
+  }
```

Comment:
We could also discuss just removing the blocking variable card for now since it's unclear on how it should work 🤔